### PR TITLE
test/integration: port etcd_storage_path_test.go to kube-apiserver testing

### PR DESCRIPTION
**Pull request by sttts**
This was moved out of https://github.com/kubernetes/kubernetes/pull/51685 to unblock kube-apiserver refactoring.

### CODE REVIEW

File: test/integration/etcd/etcd_storage_path_test.go [hunk 1]
```diff
@@ -671,122 +667,34 @@ func TestEtcdStoragePath(t *testing.T) {
...
-			tunneler, proxyTransport, err := app.CreateNodeDialer(kubeAPIServerOptions)
-			if err != nil {
-				t.Fatal(err)
-			}
-			kubeAPIServerConfig, sharedInformers, versionedInformers, _, _, err := app.CreateKubeAPIServerConfig(kubeAPIServerOptions, tunneler, proxyTransport)
-			if err != nil {
-				t.Fatal(err)
-			}
-
-			kubeAPIServerConfig.ExtraConfig.APIResourceConfigSource = &allResourceSource{} // force enable all resources
```
Comment (enj) [inline test/integration/etcd/etcd_storage_path_test.go]: How do we get this behavior back?

**Review by enj — CHANGES_REQUESTED** [review]

**Comment by sttts** [comment]
@enj do you want to take this over or do you have comments?

**Comment by enj** [comment]
@sttts my previous comments stand.  I have too much going on ATM to take this over.  https://github.com/kubernetes/kubernetes/pull/55155#discussion_r149471818 is my main concern.

**Comment by sttts** [comment]
@enj I keep this open for the moment. Maybe next year we have some more time to look at it again.

**Comment by spiffxp** [comment]
/remove-lifecycle frozen
/close
I'm closing this due to inactivity.  Please /reopen if you wish to continue working on this

**Pull request closed without merging**